### DISCUSSION TITLE
Basic help on updating an app

### DISCUSSION THREAD

**Discussion by Metal-Frog** [ScoopInstaller/Scoop#4659] [Help and Questions] [answered]
Hi,
I want to use **_scoop_** (instead of chocolatey) to manage apps used internally in my department.
So far I've created my own bucket and placed an app there. Up to here everything is fine.
This is how my manifests look like:
```
{
    "version": "1.3.2021.201",
    "description": "decript onf the app",
    "url": "path_to_zip.zip",
    "bin": "program.exe",
    "shortcuts": [
        [
            "program.exe",
            "FirstExample"
        ]
    ]
}
```

Now I thought to provide a new version of an app I just upload a new ZIP file and update the version tag in the manifest.
But if done so, nothing happens.
I think I did not understand a basic concept of scoop :).
Can someone please lead me in to the right direction? How can I easily provide a new version of an app?

Regards, Tom

**Comment by chawyehsu** [comment]
Hey @Metal-Frog glad to know you choose scoop, you can find most of the information about the project and its concept in the [Wiki](https://github.com/ScoopInstaller/Scoop/wiki). Specifically the [Bucket](https://github.com/ScoopInstaller/Scoop/wiki/Buckets) and [App Manifests](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests) pages should give some helpful information about app manifest management for you.

**Reply by Metal-Frog** [reply]
@chawyehsu: Thank you. I've read the whole wiki ...
I've started with an empty git repository for my bucket. Could this cause problems? Should I use the template?

**Reply by rashil2000** [reply]
The template is there for automating the management of manifests. If you can do that on your own, no problem :)

**Comment by Metal-Frog** [comment]
I still don't get it.
What do I have to do to tell scoop that the manifest has been updated and that there is an new version?

**Reply by niheaven** [reply]
By run `scoop update <app>` or `scoop update *` (to update all apps) in your target machine. Scoop will check manifest's json to decide whether update app(s).

**Reply by Metal-Frog** [reply]
That's what I thought - but it doesn't!
I've updated the manifest file but scoop still tells me the old version.
![image](https://user-images.githubusercontent.com/11056711/149480377-c6034130-eb76-48a9-a87a-18f5111f6a8b.png)
But in the manifest I have the version 1.3.2021.201!

**Reply by Metal-Frog** [reply]
I had to remove the bucket and add it again.
After this scoop recognized the newer version.

**Reply by Metal-Frog** [reply]
But this shouldn't be the normal procedure, should it?
I think scoop should be able to detect manifest changes in connected buckets or am I wrong?

**Comment by Metal-Frog** [comment, marked as answer]
It seems that I had a problem with git.
